LLMonFHIR/LLMonFHIRDelegate.swift

```diff
@@ -31,7 +32,7 @@ class LLMonFHIRDelegate: SpeziAppDelegate {
             CollectSamples(
                 [
                     HKClinicalType(.allergyRecord),
-                    HKClinicalType(.clinicalNoteRecord),
+//                    HKClinicalType(.clinicalNoteRecord),
```

Comment:
If you have updated the deployment target to iOS 16.4, you should be able to re-add the `.clinicalNoteRecord` data type.